### DISCUSSION TITLE
How to render text annotation on frontend?

### DISCUSSION THREAD

**Discussion by abhisheklomsh** [andreped/FP-dsa-plugin#8] [Q&A] [answered]
Posted on [histomics repo](https://github.com/DigitalSlideArchive/HistomicsTK/issues/1071) as well.

I am working on cell counting model and using DSA+Histomics for it, Can i also include a text number in the middle of the region of interest for which the nuclei segmentation boundaries are generated?

<img width="289" alt="Screenshot 2023-09-11 at 2 42 23 PM" src="https://github.com/DigitalSlideArchive/HistomicsTK/assets/13444259/c26f6d2b-5ad0-4b36-b522-b8e63edf62ac">

**Comment by andreped** [comment]
Cool application!

This sounds like a DSA/HistomicsUI-related questions, as this questions falls outside the scope of this plugin.
I see that you have already made a post at the Histomics repo, which is likely the best place to ask such this question.
If you find an answer to it, please, add an answer here as well, as it might be of interest to others.

**Comment by lomshabhishek** [comment, marked as answer]
I was able to create annotation using numpy which "draws" those characters instead of being able to pass "text" type annotation.

```
annotations.append({
                'type': 'polyline',
                'points': [[segment[0][0], segment[0][1], 0.0], [segment[1][0], segment[1][1], 0.0]],
                'closed': False,
                'fillColor': 'rgba(0,0,0,0)',
                'lineColor': color,
                'lineWidth': line_width
            })
```
